src/components/Exchange/PositionShare.tsx

```diff
@@ -34,22 +31,44 @@ function getShareURL(imageInfo, ref) {
   return url;
 }
 
-function PositionShare({ setIsPositionShareModalOpen, isPositionShareModalOpen, positionToShare, account, chainId }) {
+function PositionShare({
+  entryPrice,
+  indexToken,
+  isLong,
+  leverage,
+  markPrice,
+  pnlAfterFeesPercentage,
+  setIsPositionShareModalOpen,
+  isPositionShareModalOpen,
+  account,
+  chainId,
+}: {
```

Comment:
let's move prop types to type Props?

Reply:
I'm ok with it in general but I don't really see a reason to create `p` variable for props. It just feels redundant